### ISSUE TITLE
Question about validation

### ISSUE THREAD

**Issue by LiangbinXie** [Maclory/Deep-Iterative-Collaboration#3]
It seems that in the validating phase, it's time-consuming. Is there any way to deal with that?

**Comment by Steve-Tod** [comment]
One simple thing you can try is to reduce the size of the validation set.

**Comment by LiangbinXie** [comment]
thanks for your reply~, I findthat there is no pretrained model about HG_68_Helen.pth, can you upload it?

**Comment by Steve-Tod** [comment]
Hi, we are also using the same pretrained HourGlass model in Helen training as in CelebA training.
